docs/aks-installation.md

```diff
@@ -101,3 +109,61 @@ you should be able to see these Pods running in your cluster:
     pod "metrics-server-85c57978c6-pwzcx" deleted
     pod "tunnelfront-649ff5fb55-5lxg7" deleted
     ```
+
+## Deploy Antrea to an AKS Engine cluster
+
+Antrea is an integrated CNI of AKS Engine, and can be installed in
+`networkPolicyOnly` mode or `encap` mode to an AKS Engine cluster as part of the
+AKS Engine cluster deployment. To learn basics of AKS Engine cluster deployment,
+please refer to [AKS Engine Quickstart Guide](https://github.com/Azure/aks-engine/blob/master/docs/tutorials/quickstart.md).
+
+### Deploying Antrea in `networkPolicyOnly` mode
+
+To configure Antrea to enforce NetworkPolicies for the AKS Engine cluster,
+`"networkPolicy": "antrea"` needs to be set in `kubernetesConfig` of the AKS
+Engine cluster definition:
+
+```json
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "kubernetesConfig": {
+        "networkPolicy": "antrea"
+      }
+    }
+  }
+```
```

Comment:
what's the networkPlugin in that case? can you indicate it in the docs for reference? I assume it cannot be changed by the user

Reply:
It is Azure CNI. Sure, I can mention it.